yeat/cli.py

```diff
@@ -138,6 +140,14 @@ def get_parser(exit_on_error=True):
         default=150,
         help="target an average depth of coverage Cx when auto-downsampling; by default, C=150",
     )
+    parser.add_argument(
+        "-g",
+        "--genome-size",
+        type=int,
+        metavar="G",
+        default=0,
+        help="genome size; by default, G=0",
+    )
```

Comment:
Genome size *in base pairs*, right? If so, I'd make sure to include the units in the help message. Sometimes programs expect or support users specifying genome sizes in Kbp, Mbp, and/or Gbp.